les/clientpool.go

```diff
@@ -509,65 +622,108 @@ func (f *clientPool) getPosBalance(id enode.ID) *posBalance {
 	return balance
 }
 
-// addBalance updates the positive balance of a client.
-// If setTotal is false then the given amount is added to the balance.
-// If setTotal is true then amount represents the total amount ever added to the
-// given ID and positive balance is increased by (amount-lastTotal) while lastTotal
-// is updated to amount. This method also allows removing positive balance.
-func (f *clientPool) addBalance(id enode.ID, amount uint64, setTotal bool) {
-	f.lock.Lock()
-	defer f.lock.Unlock()
+// getPosBalanceIDs returns a lexicographically ordered list of IDs of accounts
```

Comment:
"of IDs of accounts" -> "of IDs from accounts"

Reply:
fixed